app/src/main/java/com/github/se/cyrcle/ui/addParking/attributes/AttributesPicker.kt

```diff
@@ -64,19 +66,21 @@ import com.mapbox.maps.plugin.gestures.generated.GesturesSettings
 fun AttributesPicker(
     navigationActions: NavigationActions,
     parkingViewModel: ParkingViewModel,
-    mapViewModel: MapViewModel
+    mapViewModel: MapViewModel,
+    addressViewModel: AddressViewModel
 ) {
-  val location = mapViewModel.selectedLocation.collectAsState().value!!
-  val title = remember {
-    mutableStateOf(
-        "${("%.2f").format(location.center.longitude())}, ${("%.3f").format(location.center.latitude())}")
-  }
+  val title = remember { mutableStateOf("") }
```

Comment:
Out of curiosity, why was this changed?

Reply:
The initial value of the title is irrelevant, given that as soon as the titleSuggestion is loaded from the AddressViewModel, it will get replaced. The previous value was just a placeholder @sean-prz used to avoid empty titled parkings and facilitate debugging